server/migrations/1583094753528-RemovePasswordDigest.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemovePasswordDigest1583094753528 implements MigrationInterface {
+  name = 'RemovePasswordDigest1583094753528';
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query(
+      `ALTER TABLE "user_chapter_roles" DROP CONSTRAINT "FK_b118ecdd67565f94b2a2e23d94d"`,
```

Comment:
Did you run the latest migrations? I think this was specifically updated in the latest migration 😕

Reply:
It wasn't: https://github.com/freeCodeCamp/chapter/blob/master/server/migrations/1582144372565-FixChapterRoleRelation.ts